gcp_airflow_foundations/source_class/generic_file_source.py

```diff
@@ -172,7 +172,7 @@ def get_list_of_files(self, table_config, **kwargs):
             # support replacing files with current dates
             file_list[:] = [file.replace("{{ ds }}", ds) if "{{ ds }}" in file else file for file in file_list]
             # add dir prefix to files
-            file_list[:] = [gcs_bucket_prefix + "/" + file for file in file_list]
+            file_list[:] = [gcs_bucket_prefix + file for file in file_list]
```

Comment:
what would happen if we have a real bucket prefix? 
We would need to specify it as "my_bucket/". 
Would it be cleaner if  we add "/" only if gcs_bucket_prefix is not empty?

Reply:
Yeah the options are to check if it is the empty string like we do elsewhere, or to have users ensure their prefix ends with "/". 
I guess doing the check ourselves takes away some possibility of user error, but I think we would also want to check to see if the user has passed a prefix that ends with "/" as well otherwise we could end up with a "//"

Reply:
Why not use os.path.join though?